### ISSUE TITLE
openapi expects leading "/"

### ISSUE THREAD

**Issue by AdamBien** [OpenLiberty/open-liberty#6212]
Tested with: openliberty-18.0.0.3 and openliberty-18.0.0.4  (with Java EE 8 and MP activated)

openapi requires a "/" prefix:
```java
@Produces(MediaType.APPLICATION_JSON)
@Path("/ping")
public class PingResource {

    @GET
    public Ping ping() {
        return new Ping("Enjoy Java EE 8 + MicroProfile", 42);
    }

}

@ApplicationPath("resources")
public class JAXRSConfiguration extends Application {

}
```

otherwise the openliberty will concatenate the context uri (resourcesping instead of resources/ping) with the path what will break the openapi/ui client.

Most (all?) Java EE microservices do not prepend the "/".  This should work as well:

```java
@Path("ping")
public class PingResource {
```

**Comment by kwsutter** [comment]
FYI, I just ran into this exact same problem.  I'm running with Open Liberty 18.0.0.4.  I tried the workaround of prepending the "/" to each Path endpoint and it worked.  I also tried appending a "/" to the ApplicationPath and that didn't work.  I was just trying to find a quicker workaround...  I guess the trailing "/" is ignored on the ApplicationPath.

**Comment by yeekangc** [comment]
Similar issue as #5421?

**Comment by arturdzm** [comment]
Fixed by PR #6245
